lib/testers/rule-tester.js

```diff
@@ -131,13 +131,31 @@ const DESCRIBE = Symbol("describe");
 const IT = Symbol("it");
 
 /**
- * This is `it` or `describe` if those don't exist.
+ * This is `it` default handler if `it` don't exist.
  * @this {Mocha}
  * @param {string} text - The description of the test case.
  * @param {Function} method - The logic of the test case.
  * @returns {any} Returned value of `method`.
  */
-function defaultHandler(text, method) {
+function itDefaultHandler(text, method) {
+    try {
+        return method.apply(this);
+    } catch (err) {
+        if (err instanceof assert.AssertionError) {
+            err.message += ` ${util.inspect(err.actual)} ${err.operator} ${util.inspect(err.expected)}`;
```

Comment:
I think surrounding the comparison in parentheses would be useful and make the error message more readable: ` (${util.inspect(err.actual)} ${err.operator} ${util.inspect(err.expected)})`

Reply:
Thanks for review. I've added the parentheses :-)